src/styles/header.scss

```diff
@@ -0,0 +1,65 @@
+.header {
+  max-width: 1200px;
+  display: flex;
+  font-family: Roboto, system-ui;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 500;
+  position: relative;
+  padding: 0 50px;
+  margin: 0 auto;
+
+  &__logo {
+    display: block;
+    height: 40px;
+  }
+
+  &__nav {
+    &-list {
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+      text-align: center;
+      padding: 0;
+
+      &-item {
+        list-style: none;
+        line-height: 60px;
+
+        &:not(:last-child) {
+          margin-right: 20px;
+        }
+
+        &-link {
+          display: block;
+          height: 60px;
+          text-decoration: none;
+          color: #060b35;
+          text-transform: uppercase;
+          position: relative;
+
+          &:hover {
+            color: #00acdc;
+          }
+        }
+      }
+    }
+  }
+}
+
+.is-active {
+  color: #00acdc;
+
+  &::after {
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    content: '';
+    display: block;
+    width: 100%;
+    height: 4px;
+    background-color: #00acdc;
+    border-radius: 2px;
+  }
+}
```

Comment:
The .is-active class seems to be a state class for a navigation link. It would be better to nest this inside the &__nav &-list &-item &-link selector. This way, it's clear that this class is directly related to the navigation links.